### PR TITLE
Version 0.6.2

### PR DESCRIPTION
**Pull request by carstene1ns**
ready

### CODE REVIEW

**Comment by Ghabry** [comment]
Are there any new and relevant Android translations?

File: builds/android/metadata/ca/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+EasyRPG Player és un programari que permet jugar a jocs creats amb RPG Maker 2000 i RPG Maker 2003 (els nous motors com XP, VX, VX Ace i MV no són compatibles.
```
Comment (fdelapena) [inline builds/android/metadata/ca/full_description.txt:1]: oops, missing )

Comment (Ghabry) [reply builds/android/metadata/ca/full_description.txt:1]: high quality translation >.>

File: builds/android/app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -32,7 +32,7 @@ Você pode adicionar novos diretórios de jogos nas configurações.</string>
   <string name="choose_layout">Escolha layout</string>
   <string name="unknown_region">Região desconhecida</string>
   <string name="region_modification_failed">Falha ao tentar mudar região</string>
-  <string name="refresh">Recarregar</string>
+  <string name="refresh">Atualizar</string>
```
Comment (fdelapena) [inline builds/android/app/src/main/res/values-pt-rBR/strings.xml:35]:
```suggestion
  <string name="refresh">Actualitzar</string>
```

**Pull request merged**